flocker/node/functional/test_docker.py

```diff
@@ -138,6 +140,46 @@ def start_container(self, unit_name,
 
         return d
 
+    def test_force_remove(self):
+        """
+        ``DockerClient.remove`` can forcefully remove a running container
+        when the ``force`` parameter is set to True.
```

Comment:
Something to note about this test is that, though it does exercise the new `force` parameter, it exercises a slightly different part of its behavior than we're most interested in.  "Remove a running container" isn't quite "avoid the internal server error that sometimes happens because 'Device is busy'".  It could be that `force=True` does fix this as well as its other (documented) behavior - but we haven't demonstrated it with this test.

I skimmed the Docker issue and didn't notice the section that suggests force=True as a work-around for this.  A reference to that suggestion, as well as to the issue as a whole somewhere in this code (perhaps near the callers that pass `force=True` to `remove`) would be nice.